proto/stream_plan.proto

```diff
@@ -238,6 +240,7 @@ message StreamNode {
     LookupUnionNode lookup_union = 117;
     UnionNode union = 118;
     DeltaIndexJoinNode delta_index_join = 119;
+    HashAggNode append_only_hash_agg = 120;
```

Comment:
Since there already has an `append_only` flag in `HashAggNode`, is this necessary?